HousingSearchApi/V1/Boundary/Response/GetTenureListResponse.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+
+namespace HousingSearchApi.V1.Boundary.Response
+{
+    public class GetTenureListResponse
+    {
+        private long _total;
+
+        public List<Domain.Tenure.Tenure> Tenures { get; set; }
+
+        public GetTenureListResponse()
+        {
+            Tenures = new List<Domain.Tenure.Tenure>();
+        }
+
+        public void SetTotal(long total)
+        {
+            _total = total;
+        }
+
+        public long Total()
+        {
+            return _total;
+        }
+    }
```

Comment:
Could this be made generic, as the List type is the only specific about it?
`SearchResponse<T>`

Reply:
It can be, and I thought of it too. Just don't want to do it for this PR to not waste FE time.